django/utils/log.py

```diff
@@ -52,6 +52,12 @@
             'level': 'ERROR',
             'propagate': False,
         },
+        'django.template': {
```

Comment:
It might be better not to add this in the default config -- see https://github.com/django/django/pull/4389 for a proposed change to remove other loggers.